back/spec/acceptance/admin_publications_spec.rb

```diff
@@ -41,6 +41,7 @@
       parameter :depth, 'Filter by depth (AND)', required: false
       parameter :topics, 'Filter by topics (AND)', required: false
       parameter :areas, 'Filter by areas (AND)', required: false
+      parameter :search, 'Search text of title, description, preview, and slug (AND) ?', required: false
```

Comment:
Wasn't sure about the meaning of `AND` here, hence the `?`. Let me know what's appropriate here and I can update it

Reply:
Good question. I can't find any similar example nor explanation online. I have always assumed it is just part of an optional description, and the at the '(AND)' is indicating that the query params can be chained in the request url (but then that's always the case, I guess).

Reply:
That's what I assumed. I guess I can just copy the existing format then, it's probably accurate 😅

Reply:
I think originally (in this commit 1629b84, found by `git log -G'\(AND\)'`), it meant topics or areas combined by AND logic, not OR. And then somebody copied it to another parameter (`depth`). I think we don't need AND for `search` and `depth`.

Reply:
> but then that's always the case, I guess

Yes, indeed. In this case, we wouldn't need this AND.

Reply:
Removed